tests/contrib/subjects/test_subjects_jsonschema.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+# Copyright (C) 2021 Northwestern University.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Subject JSONSchema tests."""
+
+import pytest
+from jsonschema.exceptions import ValidationError
+
+from invenio_vocabularies.contrib.subjects.api import Subject
+
+
+@pytest.fixture(scope="module")
+def schema():
+    """Returns the schema location."""
+    return "local://subjects/subject-v1.0.0.json"
+
+
+def validates(data):
+    """Validates subject data."""
+    Subject(data).validate()
+    return True
+
+
+def fails(data):
+    """Validates affiliation data."""
+    pytest.raises(ValidationError, validates, data)
+    return True
+
+
+def test_valid_full(appctx, schema):
+    data = {
+        "$schema": schema,
+        # Should it always be either "id" or "name"? Or can it be both?
```

Comment:
can be both... even "should" be both. As far as I understood `id` is the PK in the end, but `name` is the label shown to the user (at least in affiliations).